Decide/src/main/java/DD2480/decide/CMVHandler.java

```diff
@@ -253,7 +257,20 @@ private boolean licThirteen() {
 	}
 
 	private boolean licFourteen() {
-		// TODO Auto-generated method stub
-		return false;
-	}	
+        boolean greater = false;
+        boolean lesser = false;
+
+        for (int index = 0; index < (parameters.kPts - parameters.ePts - parameters.fPts - 2); index++) {
```

Comment:
change 
for (int index = 0; index < (parameters.kPts - parameters.ePts - parameters.fPts - 2); index++) {
to this
for (int index = 0; index < (dataPoints.length - parameters.ePts - parameters.fPts - 2); index++) {